language-server/test/dotty/tools/languageserver/SignatureHelpTest.scala

```diff
@@ -173,7 +291,8 @@ class SignatureHelpTest {
   }
 
   @Test def nameBasedMatch: Unit = {
-    val signature = S("", Nil, List(List(P("", "Int"), P("", "String"))), None)
+    val nameBasedMatch = S("", Nil, List(List(P("", "Int"), P("", "String"))), None)
+    val singleMatch = S("", Nil, List(List(P("", "ProdEmpty.type"))), None)
```

Comment:
`ProdEmpty.type` doesn't seem to be very useful. Should show anything here?

```suggestion
    val singleMatch = S("", Nil, List(List(P("", "ProdEmpty.type"))), None)
```

Reply:
Thats proper function signature according to documentation and https://github.com/lampepfl/dotty/issues/15230
That's how compiler will behave and in my opinion it's better to show anything than nothing, especially if it's how compiler will behave.

Reply:
ok sure! I am fine with it, just wondering :sweat_smile: